.changeset/wet-kiwis-strive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-node': patch
+---
+
+Unsets domain attribute of default cookie configurer to set OAuth cookies as host-only
```

Comment:
These words need to be added to `.github/vale/config/vocabularies/Backstage/accept.txt`